src/app.ts

```diff
@@ -97,10 +106,12 @@ async function app() {
   );
 
   // If ig-data exists, generate an IG, otherwise, generate resources only
+  logger.info('Exporting FHIR resources...');
```

Comment:
I think here it could be more clear to say 'Exporting FHIR resources as JSON...', just so it is clear why this export step is necessary, since people might assume when we say "FHIR resources" that we already mean JSON.